azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/AbstractFunctionMojo.java

```diff
@@ -151,15 +150,13 @@ public String getFinalName() {
 
     @Nullable
     public FunctionApp getFunctionApp() throws AzureAuthFailureException {
-        try {
-            return getAzureClient().appServices().functionApps().getByResourceGroup(getResourceGroup(), getAppName());
-        } catch (AzureAuthFailureException authEx) {
-            throw authEx;
-        } catch (Exception ex) {
-            Log.debug(ex);
-            // Swallow exception for non-existing Azure Functions
-        }
-        return null;
+        final PagedList<FunctionApp> functionList = getAzureClient().appServices().functionApps().list();
+        functionList.loadAll();
```

Comment:
check whether loadAll will load all heavy properties for each app

Reply:
Change to use iterator